ios/Classes/DRM/FLTEzdrmAssetsLoaderDelegate.h

```diff
@@ -0,0 +1,17 @@
+//
```

Comment:
Please remove this comment from the top. I know it's created automatically. There's no similar header comments for other files in Better Player.

Reply:
Fixed is this commit d03cee3